src/data/restore.ts

```diff
@@ -181,13 +186,20 @@ const restoreElement = (
 
 export const restoreElements = (
   elements: ImportedDataState["elements"],
+  /** NOTE doesn't serve for reconciliation */
+  localElements: readonly ExcalidrawElement[] | null | undefined,
```

Comment:
instead of passing `null` from all the places where this function is called, lets assign default value to be `null` instead

Reply:
By design to not forget. We're already doing this for `restore()` and `restoreAppState()`. I agree it's clunky, but IMO better be safe than sorry.

Reply:
for `restore()`, we could wrap both in an object so you don't have to do `, null, null)`

Reply:
Ok let it be then 👍🏻